### PR TITLE
Sentry Integration

### LINKED ISSUE #179: Sentry
## Integrar Sentry

_label: [error trace]
<br>
assignees: [assignee_username]_

<details open> 
  <summary>
    <b>Description</b>
  </summary>
  Integrar o Sentry ao projeto
</details>

<br>

<details> 
  <summary>
    <b>Additional Information</b>
  </summary>

  Link da documentação do [Sentry](https://docs.sentry.io/platforms/javascript/guides/react/?original_referrer=https%3A%2F%2Fsentry.io%2F)
  Essa task é dependente [desta](https://github.com/Alecell/octopost/issues/177)

</details>

### PR DESCRIPTION
**Pull request by PiluVitu**
Closes #179 

<details open> 
  <summary>
    <b>Feature</b>
  </summary>

Sentry integrado com o projeto
Erros são capturados e aparecem no dashboard do Sentry  
</details>

<details open> 
  <summary>
    <b>Changelog</b>
  </summary>

- Adicionei o sentry no app.tsx
- Englobei as rotas com o sentry 
</details>

<details open> 
  <summary>
    <b>Visual evidences :framed_picture:</b>
  </summary>

image

</details>

<details open> 
  <summary>
    <b>Checklist</b>
  </summary>

  - [x] Issue linked
  - [x] Build working correctly
  - [x] Tests created
</details>

<details> 
  <summary>
    <b>Additional info</b>
  </summary>
N/A
</details>

### CODE REVIEW

**Review by hxsggsz — APPROVED** [review]

File: src/App.tsx [hunk 1]
```diff
@@ -1,18 +1,49 @@
...
+import * as Sentry from '@sentry/react';
 
 import Home from './pages/home/home';
 
 import scss from './App.module.scss';
 
 import './styles/base.scss';
 
+Sentry.init({
+  dsn: 'https://bd204ec3d8157e7eb3c2fd1c1d010610@o4506298178404352.ingest.sentry.io/4506298185154560',
```
Comment (aalmeida00) [inline src/App.tsx:21]: Isso daqui parece ser alguma env. Mover para dotenv

**Review by aalmeida00 — CHANGES_REQUESTED** [review]

File: src/App.tsx (hunk 1 above)
Comment (PiluVitu) [reply src/App.tsx:21]: É uma env de dev, vou mover pra um `.env.development`

File: package.json [hunk 2]
```diff
@@ -2,9 +2,10 @@
   "name": "octopost",
   "version": "0.1.0",
   "engines": {
-    "node": "18.18.2"
+    "node": "18.18.2",
```
Comment (Alecell) [inline package.json:5]: brabo

**Review by Alecell — APPROVED** [review]
Só me confirma uma coisa, vc testou se o Sentry ta logando?

Tipo, ta tendo logs quando a gente toma erro no front?

File: .env.example [hunk 3]
```diff
@@ -0,0 +1 @@
+SENTRY_KEY = key do sentry aki
```
Comment (aalmeida00) [inline .env.example:1]: eof + ajustar.
```suggestion
SENTRY_KEY=sentry_key

```

**Review by aalmeida00 — CHANGES_REQUESTED** [review]

**Review by aalmeida00 — APPROVED** [review]

**Pull request merged**